### ISSUE TITLE
Compile errors

### ISSUE THREAD

**Issue by licaon-kter** [orgzly/orgzly-android#138]
I'm getting these for 1.5-beta.1: https://gist.github.com/licaon-kter/2a75334caf387856ef6e27cc15a2dac3
when trying to compile either directly via Gradle or via Android Studio.

Any idea on what I'm missing?

I would like to update the F-Droid build to beta.4 but I need to first compile it locally to be sure, not having any success yet with beta.1 for starters.

**Comment by fresheed** [comment]
I had similar errors with Android project. In my case problem was related to old org-java library - version in dependency is lower than actually required. After looking at Gradle scripts I found solution - create folder "orgzly" and inside it place newest org-java and orgzly-android. With that, Android project will detect development version of  library.

**Comment by nevenz** [comment]
> After looking at Gradle scripts I found solution - create folder "orgzly" and inside it place newest org-java and orgzly-android.

Yes, that directory is used if it exists (added for easier development). If not, `org_java_version` from gradle.properties is used.

@licaon-kter I can't reproduce this, when I clone orgzly-android from an empty directory. Perhaps you have org-java/ in there and it wasn't refreshed for some time? If not, org-java version used should be 1.1-SNAPSHOT.

BTW, I'll release v1.5 in a day or two, if nothing serious pops up, if you want to wait for that. (Though it would be still good to figure this out, in case we're missing something.)

**Comment by licaon-kter** [comment]
I'll try again with a clean clone, I was trying via fdroidserver and then (using the same folder) tried via gradle or Android Studio.
